### PR TITLE
Change array of key-values to python dictionary

### LINKED ISSUE #81: json output is hard to parse
The JSON output of getInfo and getPool uses JSON Arrays of ["a=X", "b=Y"] instead of Objects {"a":X, "b": Y}. This makes them non trivial to parse, since the client has to dive into the string rather then letting a JSON library do all the work for it. 

Is there a reason for doing things this way?

### PR DESCRIPTION
**Pull request by nayak16**
Resolves #81

### CODE REVIEW

**Comment by yrkumar** [comment]
LGTM, but this may require a frontend change as well to make sure we don't break Autolab integration. @ymzong @jez @icanberk?

**Comment by ymzong** [comment]
@ykumar I'm on it

**Comment by ymzong** [comment]
@yrkumar  I'm on it

**Comment by yrkumar** [comment]
We can merge this, but wait until @ymzong's change goes into Autolab to pull to production.

**Comment by ymzong** [comment]
@yrkumar @nayak16 Front-end fix is ready!

**Comment by nayak16** [comment]
Merged f779953
